README.md

```diff
@@ -748,6 +748,42 @@ Return an HTTP error code.
 Return an HTTP error code only for the given % of requests.
 If no error code was specified, default is 500.
 
+### Configuration file
+
+It is possible to put configuration options in a file like `.loadtestrc`. The options in the file will be used only if they are not specified in the command line. For more information about file name and supported file types, read the [confinode user manual](https://github.com/slune-org/confinode/blob/master/doc/en/usermanual.md). The expected structure of the file is the following:
+
+```yaml
```

Comment:
I would feel more comfortable with a JSON format.

Reply:
Will change it